updater/src/Updater/SyncLabels/IssueLabel.purs

```diff
@@ -82,6 +82,8 @@ labels =
   , questionLabel
   , mergeBefore0_14Label
   , fixBefore0_14Label
+  , mergeBefore0_15Label
+  , fixBefore0_15Label
```

Comment:
We used `merge` on PRs and `fix` on issues before, but we could probably just have a `purs-0.14` label and a `purs-0.15` label and so on to avoid doubling up each release. What do you think?

Reply:
What about remapping the '0.14' labels to the same thing?

Reply:
Done.